packages/components/src/schema/components/combobox.ts

```diff
@@ -0,0 +1,62 @@
+import type { Generic } from 'adopted-style-sheets';
+
+import type {
+	PropDisabled,
+	PropHideError,
+	PropHideLabel,
+	PropId,
+	PropLabelWithExpertSlot,
+	PropMsg,
+	PropName,
+	PropRequired,
+	PropSyncValueBySelector,
+	PropTouched,
+} from '../props';
+import type { InputTypeOnDefault, KoliBriHorizontalIcons, Stringified } from '../types';
+
+type RequiredProps = { options: string[] };
+type OptionalProps = {
+	accessKey: string;
+	error: string;
+	hint: string;
+	icons: Stringified<KoliBriHorizontalIcons>;
+	on: InputTypeOnDefault;
+	tabIndex: number;
+	value: string;
+} & PropDisabled &
+	PropHideError &
+	PropHideLabel &
+	PropLabelWithExpertSlot &
+	PropMsg &
+	PropName &
+	PropRequired &
+	PropSyncValueBySelector &
+	PropTouched;
+
+type RequiredStates = {
+	hasValue: boolean;
+	options: string[];
```

Comment:
Bitte verwendet statt `options` das Property `suggestions`. Dafür gibt es auch schon ein Type in Schema